app/vmui/packages/vmui/src/setupProxy.js

```diff
@@ -0,0 +1,11 @@
+const { createProxyMiddleware } = require('http-proxy-middleware');
+
+module.exports = function (app) {
+  app.use(
+    "/api",
+    createProxyMiddleware({
+      target: "http://localhost:8490/api",
+      changeOrigin: true,
+    })
+  );
+};
```

Comment:
Hardcoded my port, not sure how to abstract it, and not sure anyone else is going to use this besides me.